pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,198 @@
+```html
+<!doctype html>
+    <head>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+        async
+        src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+        href="https://cdn.ampproject.org/amp-story-player-v0.css"
+        rel="stylesheet"
+        type="text/css"
+    />
+    <style>
+        header {
+        height: 8vh;
+        width: 100vw;
+        display: flex;
+        justify-content: space-around;
+        color: #545454;
+        background-color: #DDB556;
+        margin-bottom: 3vh;
+        }
+        .content {
+        display: flex;
+        justify-content: space-around;
+        width: 100vw;
+        }
+    </style>
+    </head>
+    <body>
+    <header>
+        <h1>
+        Page Header
+        </h1>
+    </header>
+    <h1>
+        Article Title
+    </h1>
+    <p>
+        Doggo ipsum smol wow very biscit length boy, doing me a frighten.  Borking doggo doggo heckin dat tungg tho, heckin good boys. Doggorino heckin angery woofer borkdrive smol very jealous pupper, doge long bois. Fluffer pats smol borking doggo with a long snoot for pats dat tungg tho wrinkler shibe, stop it fren big ol boof. Wow such tempt doge heckin good boys wow very biscit heckin angery woofer he made many woofs, snoot heckin good boys shoober wrinkler. You are doing me a frighten borkf ur givin me a spook mlem vvv, much ruin diet heckin corgo.
+    </p>
+    <div class="content">
+        <amp-story-player style="width: 360px; height: 600px;">
+        <a
+        href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+        style="--story-player-poster: url('https://amp.dev/static/samples/img/story_dog2_portrait.jpg')"
+        >
+        Stories in AMP - Hello World
+        </a>
+    </amp-story-player>
+    </div>
+    <p>
+        Such treat big ol pupper. Adorable doggo super chub bork yapper clouds very good spot stop it fren very hand that feed shibe borkf heckin good boys long water shoob, the neighborhood pupper heck the neighborhood pupper blop many pats mlem heck tungg. noodle horse. Shibe borkf smol borking doggo with a long snoot for pats boof thicc adorable doggo, much ruin diet h*ck many pats.
+    </p>
+    </body>
+</html>
+```
+[/example]
+
+## Page requirements
+
+Displaying an AMP story in a non-AMP page requires the use of the [`amp-story-player`](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) element. 
+
+### Import scripts
+
+Include the two required scripts in the head of your document:
+
+```html
+<script async src="https://cdn.ampproject.org/amp-story-player-v0.js">
+<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+```
+
+The first script imports the logic for the player and the second one sets the default styling.  
+
+### Define HTML element
+
+Include the `<amp-story-player>` element inside the document `body`. 
```

Comment:
I still find this section confusing. This gives the impression you need to include an empty element.

Reply:
Enumerate the steps instead. Or simply ommit this one.